docs/0000-example-extension.md

```diff
@@ -1,6 +1,7 @@
 # OCFL Community Extension 0000: Example Extension
 
   * Authors: A Person, Other Person
+  * OCFL Version: 1.0
```

Comment:
Since we have an obsoletes and an obsoleted by, should this be a minimum required OCFL version value?

Reply:
Done